admintools/middleware.py

```diff
@@ -23,6 +23,12 @@ def process_request(self, request):
                     emulated_user = get_user_by_id(user_id)
                 elif username:
                     emulated_user = get_user_by_username(username)
-                if emulated_user:
+                if emulated_user and not emulated_user.profile.is_company_officer:
```

Comment:
let's do some refactoring

move `_can_emulate_another_user` to `htk.admintools.utils`

instead of passing in a `request` just pass in the `request.user` and inside the function check `if user.is_authenticated()`

then, add a new function, and replace the `if` condition check here

```
def can_emulate_user(original_user, targeted_user):
    can_emulate = original_user and targeted_user and not targeted_user.profile.is_company_officer
    return can_emulate
```